src/app.js

```diff
@@ -418,10 +421,18 @@ var run = function () {
     }
   })
 
+  // convert old browser-solidity
```

Comment:
Do we really need to carry over this single config option? Probably not such a big deal.